classes/Entities/CourseCategory.php

```diff
@@ -2,7 +2,28 @@
 
 namespace Avado\MoodleAbstractionLibrary\Entities;
 
+use Avado\MoodleAbstractionLibrary\Entities\BaseModel;
+use local\helpdesk\Entities\HelpdeskProduct;
+use local\helpdesk\Entities\HelpdeskProductCourses;
+
 class CourseCategory extends BaseModel
 {
     protected $table = 'course_categories';
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function subCategory(){
```

Comment:
Are we not referencing the parent here? 

If so this would be a hasOne relationship with a name of 'parent' or something similar.

Reply:
Yes we are.
Will change it

Reply:
Re-having a look at the code, I have removed the subCategory function as I do not think it is necessary because we have the category function in Course.php on line 28, doing the same I wanted this one to do

Reply:
And you're not trying to find the parent category of a category?